scipy/integrate/_ivp/ivp.py

```diff
@@ -188,9 +188,10 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
         options is determined by `vectorized` argument (see below). The
         vectorized implementation allows a faster approximation of the Jacobian
         by finite differences (required for stiff solvers).
-    t_span : 2-tuple of floats
+    t_span : 2-member iterable
```

Comment:
Would it make more sense to use `sequence` instead of `iterable`? I believe the former implies a guaranteed *ordering*, and is also more commonly used for argument types I think.

Reply:
@tylerjereddy I just did a bit of research on that matter and you're 100% correct. I just applied these changes on the latest commit.
Thank you a lot for your feedback!